cmd/dagger/config.go

```diff
@@ -359,7 +359,7 @@ func (c configSubcmd) Command() *cobra.Command {
 	if c.SetCmd != nil {
 		setCmd := &cobra.Command{
 			// TODO: make these specific based on the parent
-			Use:     "set",
+			Use:     "set [options]",
```

Comment:
```suggestion
			Use:     "set [options] <pattern>...",
```

Reply:
The `...` is missing, and on `add` and `remove` too. Can also revert here and add on the other PR since you're making a general correction on the usages there.

Reply:
My bad, reverting the `<pattern>...` right now to fix on the general correction PR